Src/AutoNSubstitute/NSubstituteVirtualMethodsCommand.cs

```diff
@@ -172,40 +174,50 @@ public ISpecimenContext Context
                 get { return context; }
             }
 
-            public static void ReturnsForAnyArgs<T>(T value, Func<CallInfo, T> returnThis)
```

Comment:
The class is a private nested class, therefore I don't think it's a breaking change.